src/imageops/sample.rs

```diff
@@ -875,6 +887,29 @@ mod tests {
         b.bytes = 800 * 800 * 3 + 200 * 200 * 3;
     }
 
+    #[test]
+    #[cfg(feature = "png")]
+    fn test_resize_same_size() {
+        use std::path::Path;
+        let img = crate::open(&Path::new("./examples/fractal.png")).unwrap();
+        let resize = img.resize(img.width(), img.height(), FilterType::Triangle);
+        assert!(img.pixels().eq(resize.pixels()))
```

Comment:
This would be a bit cleaner as `assert!(img.pixels() == resize.pixels());` or `assert_eq!(img.pixels(), resize.pixels());`

Reply:
It actually cant be that, iterators don't implement cmp::Eq, whereas eq is [part of the iterator trait](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.eq)